docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -145,15 +145,15 @@ creating a :file:`pyproject.toml` file containing, at least:
 To understand the :file:`setup.py` more fully look at the official `setuptools
 documentation`_. To compile the extension for use in the current directory use:
```

Comment:
Looks like I missed this broken place…
@scoder I included the fix in #6548 since it already touches something 3 lines above.